test/system/helpers.bash

```diff
@@ -847,6 +847,21 @@ function _ensure_container_running() {
     die "Timed out waiting for container $1 to enter state running=$2"
 }
 
+# Return the config digest of an image in containers-storage.
+# The input can be a named reference, or an @imageID (including shorter imageID prefixes)
+# Historically, the image ID was a good indicator of “the same” image;
+# with zstd:chunked, the same image might have different IDs depending on whether
+# creating layers happened based on the TOC (and per-file operations) or the full layer tarball
+function image_config_digest() {
+    echo "image_config_digest $1" >&2
+    local config=$(skopeo inspect --raw --config "containers-storage:$1")
```

Comment:
I thought `bats` is running with `-x`, yet this never aborts a test.

Reply:
oh, this one is evil. It's the `local`. It completely hides errors. Just remove it. Hashtag Bash Idiosyncrasies

Reply:
Oh wow, I _never_ would have guessed that.